site/en/r2/tutorials/load_data/csv.ipynb

```diff
@@ -0,0 +1,933 @@
+{
```

Comment:
<p>Please delete this sentence: "In production, you'd want to actually get the output."</p>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/docs/pull/487/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='60'/>